config/clusteroperator/aggregated_clusteroperator.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
+  name: aggregated
+spec: {}
+status:
+  versions:
+  - name: operator
+    version: "0.0.1-snapshot"
+  relatedObjects:
+  - group: ''
+    name: openshift-platform-operators
+    resource: namespaces
+  - group: platform.openshift.io
+    name: ""
+    resource: platformoperators
+  - group: core.rukpak.io
+    name: ""
+    resource: bundles
+  - group: core.rukpak.io
+    name: ""
+    resource: bundledeployments
```

Comment:
Prior to GA, I think we should separate out the rukpak stuff from the PO stuff. At the end of the day, PO will depend on rukpak, not provide it.

But this is fine in the mean time.

Reply:
Yup +1.